pylint/plugins/hass_constructor.py

```diff
@@ -22,7 +22,7 @@ class HassConstructorFormatChecker(BaseChecker):  # type: ignore[misc]
     }
     options = ()
 
-    def visit_functiondef(self, node: FunctionDef) -> None:
+    def visit_functiondef(self, node: nodes.FunctionDef) -> None:
```

Comment:
Why was this change needed?

Reply:
Based on the feedback in https://github.com/home-assistant/core/pull/56487#discussion_r713233925 I've decided to change it in all pylint plugins to have the same style everywhere: https://github.com/home-assistant/core/pull/56487/commits/9f9ad27a7146e45246f5bc6ac27b7bf195d1a475